examples/lux_core_engine/LuxCoreEngineScene.cc

```diff
@@ -0,0 +1,448 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include "LuxCoreEngineScene.hh"
+
+using namespace ignition;
+using namespace rendering;
+
+//////////////////////////////////////////////////
+void LuxCoreEngineScene::LogHandler(const char *msg)
```

Comment:
unused - do this here and below (if there is any other unused argument)

```suggestion
void LuxCoreEngineScene::LogHandler(const char */*msg*/)
```